Tests/Marketplace/marketplace_services.py

```diff
@@ -1715,9 +1715,14 @@ def filter_release_notes_by_entities_display_name(self, release_notes, modified_
             if rn_header not in release_notes:
                 continue
 
-            # Filters the RN entries by the entity display name
-            display_names = [list(entity.values())[0]['display_name'] for entity in entities_data]
-            filtered_release_notes_entries = self.filter_entries_by_display_name(release_notes, display_names, rn_header)
+            # Filters the RN entries by the entity display name or name
+            display_names_and_names = []
+            for entity in entities_data:
+                # Since some RN's have the name and not the display name, we check both
+                display_names_and_names.extend([list(entity.values())[0]['display_name'],
+                                                list(entity.values())[0]['name']])
+            filtered_release_notes_entries = self.filter_entries_by_display_name(release_notes, display_names_and_names,
```

Comment:
Are you sure it's neccesarry? because the display name in the id-set creating uses the same `get_display_name` function as in release notes creating, and for most entities this function takes the `name` of the entity for the `display_name`.

Reply:
you are right - i had an issue in the testing files